python/BuiltInExample1.py

```diff
@@ -0,0 +1,121 @@
+tTKPersistenceDiagram1.InputOffsetField = [None, '']
+
+# create a new 'Threshold'
+diagonal = Threshold(Input=tTKPersistenceDiagram1)
+diagonal.Scalars = ['CELLS', 'PairIdentifier']
+diagonal.ThresholdRange = [-1.0, -0.1]
+
+# create a new 'Extract Surface'
+extractSurface1 = ExtractSurface(Input=diagonal)
+
+# create a new 'Tube'
+tube1 = Tube(Input=extractSurface1)
+tube1.Scalars = ['POINTS', 'CriticalType']
+tube1.Vectors = [None, '']
+tube1.Radius = 0.02
+
+# create a new 'Threshold'
+persistencePairs = Threshold(Input=tTKPersistenceDiagram1)
+persistencePairs.Scalars = ['CELLS', 'PairIdentifier']
+persistencePairs.ThresholdRange = [-0.1, 957.0]
+
+# create a new 'Threshold'
+persistenceThreshold = Threshold(Input=persistencePairs)
+persistenceThreshold.Scalars = ['CELLS', 'Persistence']
+persistenceThreshold.ThresholdRange = [0.02, 1.0]
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints1 = TTKIcospheresFromPoints(Input=persistenceThreshold)
+tTKIcospheresFromPoints1.Radius = 0.025
+
+# create a new 'Extract Surface'
+extractSurface2 = ExtractSurface(Input=persistenceThreshold)
+
+# create a new 'Tube'
+tube2 = Tube(Input=extractSurface2)
+tube2.Scalars = ['POINTS', 'CriticalType']
+tube2.Vectors = [None, '']
+tube2.Radius = 0.015
+
+# create a new 'TTK PersistenceCurve'
+tTKPersistenceCurve1 = TTKPersistenceCurve(Input=tetrahedralize1)
+tTKPersistenceCurve1.ScalarField = ['POINTS', 'myVorticity']
+tTKPersistenceCurve1.InputOffsetField = ['POINTS', '']
+
+# create a new 'TTK TopologicalSimplification'
+tTKTopologicalSimplification1 = TTKTopologicalSimplification(Domain=tetrahedralize1,
+    Constraints=tTKIcospheresFromPoints1)
+tTKTopologicalSimplification1.ScalarField = ['POINTS', 'myVorticity']
+tTKTopologicalSimplification1.InputOffsetField = [None, '']
+tTKTopologicalSimplification1.VertexIdentifierField = [None, '']
+
+# create a new 'Warp By Scalar'
+warpByScalar1 = WarpByScalar(Input=tTKTopologicalSimplification1)
+warpByScalar1.Scalars = ['POINTS', 'myVorticity']
+warpByScalar1.ScaleFactor = 300.0
+warpByScalar1.UseNormal = 1
+
+# create a new 'TTK ScalarFieldCriticalPoints'
+tTKScalarFieldCriticalPoints1 = TTKScalarFieldCriticalPoints(Input=warpByScalar1)
+tTKScalarFieldCriticalPoints1.ScalarField = ['POINTS', 'myVorticity']
+tTKScalarFieldCriticalPoints1.InputOffsetField = ['POINTS', 'OutputOffsetScalarField']
+
+# create a new 'TTK IcospheresFromPoints'
+tTKIcospheresFromPoints2 = TTKIcospheresFromPoints(Input=tTKScalarFieldCriticalPoints1)
+tTKIcospheresFromPoints2.Radius = 5.0
+
+# create a new 'Threshold'
+minima = Threshold(Input=tTKIcospheresFromPoints2)
+minima.Scalars = ['POINTS', 'CriticalType']
+minima.ThresholdRange = [0.0, 0.1]
+
+# create a new 'Threshold'
+maxima = Threshold(Input=tTKIcospheresFromPoints2)
+maxima.Scalars = ['POINTS', 'CriticalType']
+maxima.ThresholdRange = [3.0, 3.0]
+
+# save the output
+SaveData('minima.vtu', minima)
```

Comment:
for a batch mode usage, I'd recommend to store the raw output data instead of graphical primivites.
in this example, points instead of spheres.
this is more convenient and generic for possible post-processing.

you may want to use `tTKScalarFieldCriticalPoints1` directly (or subsets, if so, you may need to use `MaskPoints` prior to thresholding).
also, please store that in `csv` file format, which is more convenient for a batch mode python noobie